pr_agent/algo/ai_handlers/litellm_ai_handler.py

```diff
@@ -10,7 +10,16 @@
 from pr_agent.log import get_logger
 
 OPENAI_RETRIES = 5
-
+import os
+import json
```

Comment:
**Suggestion:** Add a check for OLLAMA.API_BASE in the __init__ method to handle Ollama API base configuration. [enhancement, importance: 7]
```suggestion
OPENAI_RETRIES = 5
SAVE_REQUEST_FILE = "/tmp/data.jsonl"
def append_line_to_file(new_data):
    ...
class LiteLLMAIHandler(BaseAiHandler):
    """
    if get_settings().get("OLLAMA.API_BASE", None):
        litellm.api_base = get_settings().ollama.api_base
        self.api_base = get_settings().ollama.api_base
```